tools/windows-node-installer/test/e2e/aws_test.go

```diff
@@ -76,28 +77,16 @@ func setupAWSCloudProvider(t *testing.T) {
 	// The e2e test uses Microsoft Windows Server 2019 Base with Containers image, m4.large type, and libra ssh key.
 	cloud, err := cloudprovider.CloudProviderFactory(kubeconfig, awscredentials, "default", dir,
 		imageID, instanceType, sshKey, privateKeyPath)
-	if err != nil {
-		t.Fatalf("Error obtaining aws interface object %v", err)
-	}
+	require.NoError(t, err, "Error obtaining aws interface object")
 	credentials, err := cloud.CreateWindowsVM()
-	if err != nil {
-		t.Fatalf("Error spinning up Windows VM with error %v", err)
-	}
-	if credentials == nil {
-		assert.FailNow(t, "Credentials returned are empty")
-	}
-	if len(credentials.GetPassword()) == 0 {
-		assert.FailNow(t, "Expected some password but got empty string")
-	}
-	if len(credentials.GetInstanceId()) == 0 {
-		assert.FailNow(t, "Expected some instance id but got empty string")
-	}
+	require.NoError(t, err, "Error spinning up Windows VM")
+	assert.NotEmpty(t, credentials, "Credentials returned are empty")
+	assert.NotEmpty(t, credentials.GetPassword(), "Expected some password but got empty string")
+	assert.NotEmpty(t, credentials.GetInstanceId(), "Expected some instance id but got empty string")
```

Comment:
These need to be requires as they were `assert.FailNow()` before.

Reply:
will change that.